src/Workflows/AutomaticRetryWorkflow.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+use Zenaton\Interfaces\WorkflowInterface;
+use Zenaton\Traits\Zenatonable;
+
+class AutomaticRetryWorkflow implements WorkflowInterface
+{
+    use Zenatonable;
+
+    public function handle()
+    {
+        (new TaskWithRetry())->execute();
+    }
+}
```

Comment:
Why this file? I'm not sure to understand what it explains to users

Reply:
for the record

> Gilles  12 minutes ago
yes - why do we need this - instead of a single task ?

> Antoine  9 minutes ago
Ah , you mean why use a workflow instead of a single task ? It was mostly because we do not have yet the same dashboard for single task that we have for workflow. I wanted the user to be able to see real-time progress of fail/retry.

> Gilles  8 minutes ago
:+1: ok I did not think about that